okhttp-tls/src/main/kotlin/okhttp3/tls/HeldCertificate.kt

```diff
@@ -407,11 +412,11 @@ class HeldCertificate(
 
       if (maxIntermediateCas != -1) {
         result += Extension(
-            extnID = ObjectIdentifiers.basicConstraints,
+            id = basicConstraints,
             critical = true,
-            extnValue = BasicConstraints(
+            value = BasicConstraints(
                 ca = true,
-                pathLenConstraint = 3
```

Comment:
only behavior change. This was a copy-paste bug and the test happens to be asserting that the sample value is exactly 3!